src/drain.rs

```diff
@@ -1,4 +1,5 @@
 use std::iter::{ExactSizeIterator, FusedIterator};
+use std::mem::drop;
```

Comment:
`drop` is in the prelude, we need not import it

Reply:
Oh oops. I'll remove that.

I think there's a good argument to use `.for_each(drop)` just to be super clear about what's happening. What do you think?

Reply:
I don't mind either way, we can always revisit that later!